sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+import scrapy
+from scrapy import Selector
+from scrapy.spiders import CrawlSpider
+
+
+class IvoclarVivadent(CrawlSpider):
+    name = 'ivoclar_vivadent'
+    base_url = 'https://www.ivoclarvivadent.se/'
+
+    def start_requests(self):
+        url = 'https://www.ivoclarvivadent.se/sv/skerhetsdatablad/'
+
+        yield scrapy.Request(url=url, callback=self.parse)
```

Comment:
no need for this method use `start_urls` field.

Reply:
When I use start_urls it doesn't scrap any data